### PR TITLE
Final project commit

### PR DESCRIPTION
**Pull request by z4cope**
## Description of the task to be completed
- [x] Used the [open-weather-app](https://openweathermap.org/api/geocoding-api) api for displaying countries and cities. 
- [x] Also used [open-weather-app](https://openweathermap.org/current) api for displaying each city weather details.
- [x] Responsive design.
- [x] Created a home page and displayed locations in it.
- [x] Created details page and displayed each city or country weather details inside it.
- [x] Countries and cities are searchable using the search icon at the top of the screen.
- [x] Back button is shown every time the user navigates to the details page.
- [x] Used Redux-tool kit for creating a slice and logic for each api call.
- [x] Implemented unit testing using jest for the `navbar` component and redux logic.

## How should it be tested
Clone the repository
```
git clone https://github.com/z4cope/react-weather-app
```
Navigate to the folder
```
cd dev
```
Change branch
```
git checkout dev
```
#### Thanks for reviewing my PR 😊

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,246 @@
...
+- **Develop webapp around an API**
+- **Conduct test to ensure all edge cases are covered**
+- **Create web page using given wireframe**
+- **Users can filter amount of categories displayed**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](#https://shopping-sage.vercel.app/)
```
Comment (BenMukebo) [inline README.md]: ##### **Live version**
- [x] According to the project requirement,  you should deploy the application using Heroku, Netlify, or GitHub Pages. Kindly address this 👍🏻 and make sure:
  1 The project is deployed correctly, and the home and detail page works as expected.
  2. A link to the deployed project is present in the README.

**Review by BenMukebo — CHANGES_REQUESTED** [review]
## Hi @z4cope, 

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To Highlight 🎉
- Git flow is followed. :heavy_check_mark:
- You have followed  JavaScript best practices. ✔️ 
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️
- Good presentation video. ✔️ 
- Your Design looks awesome, keep rocking :rocket: 

## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers :clinking_glasses:  and Happy coding!!! :dancers:💻  

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -1 +1,246 @@
...
+Give a ⭐️ if you like this project!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- Thanks to [Microverse](https://www.microverse.org/) for giving this opportunity to learn ...
+- Code Reviewers & Coding Partners.
+- Hat tip to anyone whose code was used.
+
```
Comment (KayLemba) [inline README.md:223]: [OPTIONAL] The README contains an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/). Please kindly add that the previous code reviewer missed this

**Review by KayLemba — APPROVED** [review]
[# STATUS: APPROVED :green_circle: :clap: 

Hello  @z4cope         , 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

HIGHLIGHTS:

- [x]  Your repo is professional 
- [x] Descriptive P.R







 








Now you can merge this branch and move on :1st_place_medal: 
Happy Coding cheers 🥇 💯 



well-1

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._]
